project-backend/project-backend.csproj

```diff
@@ -7,12 +7,14 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="AspNetCore.IQueryable.Extensions" Version="5.0.2" />
```

Comment:
This is nice, but do you use it anywhere?

Reply:
The "Include" function comes from here, and it's needed to lazy load data in queries that contain joins and the sort.